dist/index.js

```diff
@@ -9637,13 +9637,80 @@ console.log('GITHUB_WORKSPACE', GITHUB_WORKSPACE);
 const pantheonDeploy = (() => {
 
     const init = ({
+        prState,
```

Comment:
You are using prState here but pullRequest further down. Could we maybe use pullRequestState?